docs/wfe.rst

```diff
@@ -0,0 +1,216 @@
+Representing sources of wavefront error
+=======================================
+
+When used for Fraunhofer-domain propagation, POPPY supports expressing aberrations as if they were applied in a pupil plane. Currently, there is the `~poppy.ZernikeWFE` optical element to represent wavefront error as a combination of Zernike terms, and the `~poppy.ParameterizedWFE` optical element that offers the ability to specify basis sets other than Zernikes.
+
+ZernikeWFE
+----------
+
+For use in POPPY calculations, the `~poppy.ZernikeWFE` analytic optical element provides a way to introduce wavefront error as a combination of Zernike terms. The Zernike terms are both ordered in and normalized in the convention of Noll 1976. [#noll1976]_ This means that the polynomial terms carry a normalization factor to make :math:`\int_0^{2\pi} \int_0^1 Z_j^2\,\rho\,d\rho\,d\theta = \pi.` Additionally, the standard deviation of the optical path difference over the disk for any given term will be 1.0 meters. (See :ref:`zernike_normalization` for details on how this might impact your calculations.)
```

Comment:
First phrase 'For use in POPPY calculations,' seems redundant/unnecessary. Brevity is the soul of wit, etc.